sonnydashboard/server.js

```diff
@@ -0,0 +1,99 @@
+var   express     = require('express'),
+      bodyParser  = require('body-parser'),
+      path        = require('path'),
+      io          = require('socket.io'),
+      mongoose    = require('mongoose')
+      app         = express();
+
+app.use(bodyParser.urlencoded({extended: true}));
+
+app.set('view engine', 'ejs');
+app.set('views', path.join(__dirname, './views'));
+
+//Routes
+
+app.get('/', function(req, res) {
+  Animal.find({}, function(err, animals) {
+    if(err) {
+      console.log('something went wrong');
```

Comment:
In the long run, we want to have error handling happen here. For now a console.log will work, but eventually we have to let the client know if something got messed up.